src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -375,10 +375,9 @@ private SuggestedActionSet InlineActions(SuggestedActionSet actionSet)
                     // See https://github.com/dotnet/roslyn/issues/29589
                     const bool includeSuppressionFixes = true;
 
-                    var fixes = Task.Run(
+                    var fixes = this.ThreadingContext.JoinableTaskFactory.Run(
                         () => _owner._codeFixService.GetFixesAsync(
```

Comment:
❗ This needs to explicitly switch to the background thread, otherwise `GetFixesAsync` will now be invoked on the UI thread.

Reply:
done.